app/src/main/res/layout/fragment_detailed_item.xml

```diff
@@ -48,6 +51,14 @@
             android:textSize="18sp"
             tools:text="Here goes the description of the item." />
 
+        <TextView
+            android:id="@+id/itemPostedBy"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textSize="18sp"
+            android:text="@{username}"
+            tools:text="Posted by TestUser" />
```

Comment:
Maybe we should make this a string resource?

Reply:
It's a tools text, so I don't know if we should do that, I think we haven't done it for any of the others.

Reply:
Yeah, I guess it's fine